src/connectionController.ts

```diff
@@ -416,58 +422,55 @@ export default class ConnectionController {
     void vscode.window.showInformationMessage('MongoDB connection successful.');
 
     this._activeDataService = newDataService;
-    this._activeConnectionModel = connectionModel;
+    this._activeConnectionModel = await convertConnectionInfoToModel({ connectionOptions });
```

Comment:
Do we still need this? Unless I'm missing something looks like we can pull everything from the connection options for the connection, maybe we can remove it.

Reply:
Also leftover :) thx for noticing it!